docs/conf.py

```diff
@@ -39,8 +39,8 @@
 project = "ROCm Documentation"
 author = "Advanced Micro Devices, Inc."
 copyright = "Copyright (c) 2023-2024 Advanced Micro Devices, Inc. All rights reserved."
```

Comment:
@samjwu Please update the copyright to 2024. Thanks so much.

Reply:
Good catch! Fixed